client/components/shared/footer/Footer.tsx

```diff
@@ -255,19 +255,21 @@ export const Footer = () => {
 									</ul>
 								))}
 
-								<div css={supportStyles}>
-									<div css={supportTitleStyles}>
-										Support the&nbsp;Guardian
+								{!hideSupport && (
```

Comment:
minor but could use a `showSupport` var instead and avoid a couple of negations

Reply:
Hey, I could do yeah, although the variable would require negation in its declaration. 

I went for `hideSupport` as the prop name so that you can use the footer throughout MMA without having to pass through an additional prop, so hiding the CTA's is the exception. For context this footer component is shared between the help centre and manage my account.

Reply:
> use the footer throughout MMA without having to pass through an additional prop
fair 👍